packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -72,37 +72,37 @@ export async function errorHandling(errorDetails: Error | string, label?: string
             if (imperativeError.mDetails.additionalDetails) {
                 const tokenError: string = imperativeError.mDetails.additionalDetails;
                 if (tokenError.includes("Token is not valid or expired.")) {
-                    if (isTheia()) {
+                    if (this.isTheia()) {
```

Comment:
I think this change caused an error because `this` was undefined?
Maybe it's ok to leave as-is for now because I think the error was fixed in main branch which will eventually make its way into next branch 😋